lbrynet/conf.py

```diff
@@ -173,6 +173,7 @@ def _convert_value(value):
     'API_ADDRESS': 'lbryapi',
     'APP_NAME': APP_NAME,
     'BLOBFILES_DIR': 'blobfiles',
+    'COMMENT_SERVER': 'http://18.233.233.111:2903/api',
```

Comment:
@nikooo777 can we get a domain name going for this server? comments.lbry.io?

Reply:
you got it. comments.lbry.io